src/Confluent.Kafka/Consumer.cs

```diff
@@ -210,7 +212,7 @@ private static byte[] ValueAsByteArray(rd_kafka_message msg)
         ///     <see cref="Confluent.Kafka.Consumer{TKey,TValue}.Assign(TopicPartition)" />
         ///     or implicitly.
         /// </summary>
-        public List<TopicPartition> Assignment
+        public List<TopicPartition> AssignedPartitions
```

Comment:
I dont feel this change is really needed, it is splitting hairs.

Reply:
the point of the PR is to differentiate between these two concepts:
1. consumer group partition assignment
2. the set of partitions the consumer is assigned to read from (which can be different from 1 in librdkafka, but not the java client).

without this change, the rest of the PR also doesn't make sense.

Reply:
But in my book Assignment is what was Assign():ed. Follow the logic here we should also rename Assign() to AssignPartitions() - which I don't think adds any value either.

Yes, there is a consumer group assignment, but we don't have an API to get those, so I still think we should keep the Assignment =~ Assign pairing.

Reply:
there is an API to provide the consumer group assignment - it's the callback, and whatever API we end up for that will use the terminology Assignment somewhere in it, which you're proposing we should overload with a different meaning here.

I don't think the logic you point out re naming holds.